common/shlibs

```diff
@@ -1739,8 +1739,8 @@ libportmidi.so portmidi-217_1
 libsox.so.3 sox-14.4.2_1
 libsoxr.so.0 libsoxr-0.1.2_1
 libsoxr-lsr.so.0 libsoxr-0.1.2_1
-libdumb.so.2 dumb-2.0.3_1
-libaldmb.so.2 dumb-2.0.3_1
+libdumb.so.2 dumb-2.0.3_2
+libaldmb.so.2 dumb-2.0.3_2
```

Comment:
Did this shared library change? If not, this entry shouldn't be changed, since it tells the system the minimum version of `dumb` that contains the library.